ng-dev/release/info/cli.ts

```diff
@@ -6,31 +6,61 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {CommandModule} from 'yargs';
+import {Arguments, Argv, CommandModule} from 'yargs';
 
 import {GitClient} from '../../utils/git/git-client';
-import {assertValidReleaseConfig} from '../config/index';
+import {assertValidReleaseConfig, NpmPackage} from '../config/index';
 import {fetchActiveReleaseTrains} from '../versioning/active-release-trains';
 import {printActiveReleaseTrains} from '../versioning/print-active-trains';
 import {getNextBranchName, ReleaseRepoWithApi} from '../versioning';
 import {getConfig} from '../../utils/config';
 
+/**
+ * Type describing the JSON output of this command.
+ *
+ * @important When changing this, make sure the release action
+ *   invocation is updated as well.
+ */
+export type ReleaseInfoJsonStdout = {npmPackages: NpmPackage[]};
+
+/** Command line options for printing release information. */
+export interface ReleaseInfoOptions {
+  json: boolean;
+}
+
+/** Yargs command builder for the `ng-dev release info` command. */
+function builder(argv: Argv): Argv<ReleaseInfoOptions> {
+  return argv.option('json', {
+    type: 'boolean',
+    description: 'Whether information should be written as JSON to stdout.',
+    default: false,
+  });
+}
+
 /** Yargs command handler for printing release information. */
-async function handler() {
+async function handler(argv: Arguments<ReleaseInfoOptions>) {
+  const config = getConfig();
+  assertValidReleaseConfig(config);
+
+  // If JSON output is requested, print the information as JSON to stdout.
+  if (argv.json) {
```

Comment:
I am not quite sure how this makes sense to list within this command.  The JSON information being returned is entirely different than what is printed without the `--json` flag.

Reply:
Yeah, I wasn't entirely sure about this, but given this is just a layer of communication for us, and it being strictly typed, I chose that way.

I initially considered another command like ng-dev release is-experimental.. but that is not very future-proof